src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,8 @@
+package seedu.addressbook.data.person;
+
+public interface Printable {
+	/*
```

Comment:
header comments should start with `/**`, the double asterisks have meaning, do look it up!